File: chat/src/main/java/com/difft/android/chat/scribbles/RemoveEmojiTextFilter.java
package com.difft.android.chat.scribbles;

import androidx.annotation.NonNull;

import org.signal.imageeditor.core.HiddenEditText;
import com.difft.android.chat.components.emoji.EmojiUtil;

class RemoveEmojiTextFilter implements HiddenEditText.TextFilter {
  @Override
  public String filter(@NonNull String text) {
    return EmojiUtil.stripEmoji(text);
  }
}

File: chat/src/main/java/com/difft/android/chat/scribbles/RemoveEmojiTextFilter.kt
package com.difft.android.chat.scribbles

import com.difft.android.chat.components.emoji.EmojiUtil
import com.difft.android.imageeditor.core.HiddenEditText

internal class RemoveEmojiTextFilter : HiddenEditText.TextFilter {
    override fun filter(text: String): String? = EmojiUtil.stripEmoji(text)
}
